src/main/java/ai/rev/speechtotext/ApiInterface.java

```diff
@@ -54,4 +55,13 @@
           @QueryMap Map<String, String> query,
           @HeaderMap Map<String, String> contentType
   );
+
+  @POST("vocabularies")
```

Comment:
Since the CV API is a different api, can we define the methods in their own interface?